src/main/java/entities/Review.java

```diff
@@ -1,71 +1,40 @@
 package entities;
 
-public class Review implements Savable{
+public class Review {
     private int rating;
     private String comment;
```

Comment:
if the comment should not be changed later, then `private final String comment` would be better